spec/models/rent_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe Rent do
+  subject(:rent) { build(:rent) }
+
+  it { is_expected.to validate_presence_of(:start_date) }
+  it { is_expected.to validate_presence_of(:end_date) }
+  it { is_expected.to validate_presence_of(:book) }
+  it { is_expected.to validate_presence_of(:user) }
+
+  describe '#save' do
```

Comment:
same i commentes in the other PR, this is not necessary here, probably it would be better to do this kind of validations in the controller of this models when the time comes